benchmark/infrastructure/BenchmarkMain.cpp

```diff
@@ -149,16 +165,7 @@ int main(int argc, char** argv) {
 
   // Print all the available configuration options, if wanted.
   if (vm.count("configuration-options")) {
-    std::ranges::for_each(
-        BenchmarkRegister::getAllRegisteredBenchmarks(),
-        [](const BenchmarkInterface* bench) {
-          std::cerr << createCategoryTitle(absl::StrCat("Benchmark class '",
-                                                        bench->name(), "'"))
-                    << "\n"
-                    << bench->getConfigManager().printConfigurationDoc(true)
-                    << "\n\n";
-        });
-    exit(0);
+    printConfigurationOptions();
```

Comment:
Call this function `printConfigurationOptionsAndExit` and mark it `[[noreturn]] ` as suggested by sonar.

Reply:
Done, see [0fa1702](https://github.com/ad-freiburg/qlever/pull/1169/commits/0fa17021c682924ec9b8491b60ca145521a882a5).